tests/integration/component.go

```diff
@@ -373,4 +374,43 @@ func componentTests(args ...string) {
 		*/
 	})
 
+	Context("Test odo push with --now flag during creation", func() {
+		var originalDir string
+
+		JustBeforeEach(func() {
+			context = helper.CreateNewContext()
+			project = helper.CreateRandProject()
+			originalDir = helper.Getwd()
+			helper.Chdir(context)
+		})
+
+		JustAfterEach(func() {
+			helper.Chdir(originalDir)
+			helper.DeleteProject(project)
+			helper.DeleteDir(context)
+		})
+
+		It("should successfully create config and push code in one create command with --now", func() {
+			appName := "nodejs-create-now-test"
```

Comment:
Does it really needed ? i mean to say why assigning a value which can be directly used like a simple name ```nodejsApp```, similarly component name would be ```nodejsComp```. Anyway the spec description make sense of what should be there in the steps then IMO it is unnecessary to over naming the values and assigning it variables for use in steps.